core/plugins/org.polarsys.capella.core.data.core.properties/src/org/polarsys/capella/core/data/core/properties/fields/BasicElementGroup.java

```diff
@@ -145,5 +146,9 @@ public void enableSummaryField(boolean enabled) {
   public void setEnabled(boolean enabled) {
     LockHelper.getInstance().enable(nameTextField, enabled);
     LockHelper.getInstance().enable(summaryTextField, enabled);
+	if (enabled && semanticElement instanceof CapellaElement) {
+		// Refresh widgets from semantic element in case of remote update
+		loadData(semanticElement);
```

Comment:
The setEnabled/enableButton/enabledxxx methods goal is to enable or disable the properties ui elements, it must not call load method to modify the internal state of the view. 
It shall have somewhere in Team for Capella that is allowing to reload the property .
Maybe in com.thalesgroup.mde.melody.collab.ui.lock.listeners.CDOReadOnlySectionHandler, maybe in notifyIsReleased

Reply:
Hi Philippe,
I will look into it, I made the change on the widget level because I was missing a more global way to trigger a refresh after the enablement (as it was expected to be needed only on focus change). I may have missed something.